sdk/servicebus/azure-servicebus/azure/servicebus/management/_generated/aio/_service_bus_management_client_async.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Any, Optional
+
+from azure.core import AsyncPipelineClient
+from msrest import Deserializer, Serializer
+
+from ._configuration_async import ServiceBusManagementClientConfiguration
+from .operations_async import QueueOperations
+from .operations_async import ServiceBusManagementClientOperationsMixin
+from .. import models
+
+
+class ServiceBusManagementClient(ServiceBusManagementClientOperationsMixin):
+    """Azure Service Bus client for managing Queues, Topics, and Subscriptions.
```

Comment:
```suggestion
    """The ServiceBusManagementClient class defines a high level interface for managing Queues, Topics, Subscriptions and Rules.
```

nitpick: copying some words from the normal data-plane client